k8s/helm/values.yaml

```
@@ -31,6 +31,8 @@ promtail:
       pipelineStages:
         - cri: {}
         - decolorize: {}
+    server:
+      log_level: info
```

This setting did not work for me.

But his worked:
```suggestion
    logLevel: info
```